malariagen_data/anoph/h12_params.py

```diff
@@ -22,6 +22,13 @@
     """,
 ]
 
+multi_window_size: TypeAlias = Annotated[
+    Union[window_size, dict[str, int]],
+    """
+    The size of windows (number of SNPs) used to calculate statistics within.
```

Comment:
```suggestion
    The size of windows (number of SNPs) used to calculate statistics within. Can
    be a single value, in which case the same window size will be used for all
    cohorts. Can also be a mapping from cohort identifiers to values, in case
    you need to provide different window sizes for different cohorts.
```